planning/planning_error_monitor/CMakeLists.txt

```diff
@@ -58,5 +58,6 @@ endif()
 
 ament_auto_package(
   INSTALL_TO_SHARE
-    launch
+  config
+  launch
```

Comment:
To minimize the diff.
![image](https://user-images.githubusercontent.com/31987104/165016911-ace40ffb-309b-466a-b9e8-f1c3ddfa18c1.png)

```suggestion
    config
    launch
```